### DISCUSSION TITLE
Failed to connect via rendezvous server, please try later

### DISCUSSION THREAD

**Discussion by teescripts** [rustdesk/rustdesk#3377] [General]
Just installed Rustdesk for the first time (macbook pro).
Got another person to install on windows.
He shared his ID but when i hit connect, all i get is: 
**Failed to connect via rendezvous server, please try later**

**Comment by Billy1900** [comment]
I have the same issue. I build a sever by myself. Three machines are using the same sever (client1, server1, server2). client1 can connect server1, but cannot connect to sever2. The error is Failed to connect via rendezvous server, please try later.

**Comment by F4LC0NX** [comment]
I have the same issue, I had thw client working on win 10, but suddenly I cant connect to the remote win 10. I've tried connecting using Linux deb but the same problem. I disabled both the firewalls from the remote pc and my pc. Using 1.2.3, also tried different builds
